src/popup/Router.tsx

```diff
@@ -96,10 +96,10 @@ const HomeRoute = () => {
 
     /* 
     We want to launch the extension in a new tab for a user still in the onboarding process.
-    In this particular case, open the tab with a query param that stops subsequent redirects.
+    In this particular case, open the tab if we are in the "popup" view.
     */
-    if (location.search !== "?sameTab=true") {
-      window.open(newTabHref("/?sameTab=true"));
+    if (window.innerWidth === POPUP_WIDTH) {
+      window.open(newTabHref("/"));
```

Comment:
this is slightly unrelated, but was causing a problem during work on this: 

creating a more deterministic way to figure out when to open in a new tab.